JhipsterXamarin/UnitTestJhispterXamarin/packages.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="4.4.0" targetFramework="net472" />
+  <package id="Moq" version="4.14.7" targetFramework="net472" />
+  <package id="MSTest.TestAdapter" version="2.1.1" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="2.1.1" targetFramework="net472" />
+  <package id="MvvmCross" version="7.1.2" targetFramework="net472" />
+  <package id="MvvmCross.Tests" version="7.1.2" targetFramework="net472" />
+  <package id="NUnit" version="3.12.0" targetFramework="net472" />
+  <package id="System.Console" version="4.3.1" targetFramework="net472" />
+  <package id="System.IO" version="4.3.0" targetFramework="net472" />
+  <package id="System.Net.Http" version="4.3.4" targetFramework="net472" />
```

Comment:
To resolve you problem you need to add 
  `<package id="System.Net.Http.Json" version="3.2.1" targetFramework="net472" /> `
here